packages/opc-feedback/src/opc-feedback.ts

```diff
@@ -105,14 +124,59 @@ export class OpcFeedback extends LitElement {
       crossorigin="anonymous"/>
     <link type="text/css" rel="stylesheet" href="https://unpkg.com/@patternfly/patternfly/patternfly-addons.css"
       crossorigin="anonymous"/>
+    
+    <!-- Bug Panel -->
+    <dialog id="bug-dialog" class="op-feedback__panel pf-u-mr-0 pf-u-display-block" .open="${this._openBugModal}">
+      <form class="bug-form" id="bugform">
+        <div class="pf-u-display-flex">
+          <ion-icon name="arrow-back-sharp" class="pf-u-mt-sm pf-u-font-size-lg" @click="${e => {
+            this._setModalState(true, false, false, true);
+            this._resetForm();
+          }}"></ion-icon>
```

Comment:
replace all the ion-icons with svgs

Reply:
What's is problem of using it.

It was there since the first version

Reply:
I removed it